### DISCUSSION TITLE
Only load chunks while no player online

### DISCUSSION THREAD

**Discussion by Aelux5216** [BG-Software-LLC/WildLoaders#33] [Pending Feature Requests]
I think it would be cool to be able to only load when playercount = 0 this way while we play we have the usual lag free experience then overnight our farms do their work and then when someone comes online again things are business as usual again. Some sort of config setting such as enablewhileplayersonline = true/false for example. I know mechanically it is probably a lot of effort but suggestions are suggestions after all :)

**Comment by CommandDan** [comment]
I am currently implementing the opposite for a server, but I'll further down the line include customization which could also allow for this functionality.
